iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/provisioning/ProvisioningServiceClientTests.java

```diff
@@ -86,4 +90,57 @@ public void enrollmentGroupGetAttestationMechanismSymmetricKey() throws Provisio
         assertTrue(retrievedSymmetricKeyAttestation.getPrimaryKey().length() > 0);
         assertTrue(retrievedSymmetricKeyAttestation.getSecondaryKey().length() > 0);
     }
+
+    @Test
+    public void createdEnrollmentGroupMatchesSentEnrollmentGroup() throws ProvisioningServiceClientException
+    {
+        String groupId = testPrefix + UUID.randomUUID();
+        Attestation attestation = new SymmetricKeyAttestation("", "");
```

Comment:
create an object will empty-string params?

Reply:
When we pass in null or empty symmetric keys, the service will generate a pair for us. Since the key doesn't need to be a particular value for this test, we may as well leave it blank

Reply:
Got it. Wouldn't it be better if we had a no-args constructor though? Requiring customers to pass in null/empty string in order to get service-generated symmetric keys seems unnecessary, right?

Reply:
We do have a no-args constructor, but it's implementation (for better or worse) generates some keys locally for the user. Since we don't need to generate any keys locally, I avoided that no-args constructor